modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/base-page/Header.tsx

```diff
@@ -143,6 +144,17 @@ const PageActions: React.FC<IPageActionsProps> = ({
 				{actions.map(({label, ...props}) => (
 					<ClayDropDown.Item key={label} {...props}>
 						{label}
+
+						{label === 'Static Segment' && (
```

Comment:
hey @CaioERP please check if its ok to use the same logic we discussed today

`deprecated && (...)`